crates/host/src/wasmbus/mod.rs

```diff
@@ -1101,12 +1179,70 @@ impl Lattice {
                 instances: HashMap::default(),
             });
         if let hash_map::Entry::Vacant(entry) = instances.entry(link_name.clone()) {
-            debug!(?path, "spawn provider process");
-            let child = process::Command::new(&path)
+            let id = Ulid::new();
+            let async_nats::ServerInfo { host, port, .. } = self.nats.server_info();
```

Comment:
The provider RPC settings (host, port, JWT, seed, etc) are not necessarily the same as the the CTL interface settings, or the RPC settings for the host. See https://wasmcloud.com/docs/hosts/elixir/host_configure for details

I'm okay with assuming everything is the same for now — we've hard-coded `default` for the lattice prefix too — but please file an issue to add support for different settings

Reply:
Yes, that will be fixed together with all the config TODOs